content/post/2018-02-19-thoughts-to-words-an-introduction-to-aaron-simumba-s-website.md

```diff
@@ -0,0 +1,68 @@
+
+My website is built on the `hugo`, `github`, `netlify` and `blogdown` platform. I am using the [Beautiful Hugo](https://github.com/halogenica/beautifulhugo) theme. I like this theme for the following reasons: natively it comes with support for syntax highlighting, mathjax, a big image cover for the landing page and photoswipe, embedding a gallery of images is something I wanted.
+
+![screenshot](https://user-images.githubusercontent.com/24398851/36398695-035a64fe-15d1-11e8-87b1-6a7c5ab65ad9.png)
+
+
+The theme has quite a clean design, which you can get started with a new website in minutes. Minimalistic design if you decide to strip out the images. Looking at the amount of work I did to achieve the visual effects as seen below. I must say I'm pleased and happy to call this a personal website!
+
+![website](https://user-images.githubusercontent.com/24398851/36397146-09733e62-15ca-11e8-9be1-a91244efc209.jpg)
+
+
+
+I broke and repositioned a lot of the native features to achieve the design as seen above. I let go of the avatar container on the navigation bar, re-aligned the navbar links to the right from their original left hand position. Added font awesome icons to some links. 
+
+![navbar](https://user-images.githubusercontent.com/24398851/36397152-0f79484c-15ca-11e8-9974-f8be9bef098a.png)
+
+
+For the landing page, I moved posts preview to the `blog` tab. And in addition, adjusted the big image of the landing page to cover the whole page. Resized the footer to accommodate the big front image.
+
+I love the post preview from the blog section. It gives a brief section to preview the content before delving fully into the post. From the original theme structure, this is presented on the front page if you use `/content/post/` structure. To avoid this, I changed it to `/content/blog/`. All my posts reside in the `/blog` subdirectory.
+
+![blog](https://user-images.githubusercontent.com/24398851/36397317-016df4c2-15cb-11e8-986f-b33a31689bc6.jpg)
+
+# End goal
+
+Going forward, I will use this website to document my R learning path. And occasionally anything I find remotely interesting for the future me for reference purposes. More like turning my wild thoughts to written engravings. Happy blogging...
+
+I cannot document all the small but significant changes I made. The best option would be to fork the site and have a look at the changes made. For the source to my website, you can find it [here](https://github.com/rbind/asimumba)
```

Comment:
Small thing, but maybe an ending period at the end of the sentence?

Reply:
@joongsup Thanks for pointing that out. I have rectified the missing period. On the thumbnails on images. I will have to tweak the theme to make it compatible with the aspect ratio of my images. Actually the images are there if you hover over the the blank space. Due to the theme restriction, images have to be set to a certain specified length to width ratio.